extra/discover-host

```diff
@@ -16,6 +16,7 @@ base = find_base
 version = "3.4.0"
 interfaces = []
 preserve_interfaces = false
+use_facts_endpoint = false
```

Comment:
How is this related?

Reply:
It is not, do you insist on splitting this into two PRs?